core/src/main/resources/mappings/scheduled-jobs.json

```diff
@@ -114,6 +114,19 @@
                   "message_template": {
                     "type": "object",
                     "enabled": false
+                  },
+                  "throttle_enabled": {
```

Comment:
Any specific reason you have "throttle_enabled" outside "throttle"?
e.g.
```
"throttle": {
    "properties": {
        "enabled": { "type": "boolean" },
        "value": { "type": "integer" },
        "unit": { "type": "keyword": }
    }
}
```

Reply:
We may add throttle on Monitor/Trigger level, they function as default throttle if the lower level's throttle is not defined. I think the throttle enable or not should be on action level. That's why put the "throttle_enabled" outside "throttle".

Reply:
But, will we also support "enabled" at the trigger and monitor level? Or will "enabled" only be at the action level and eventually action, trigger, monitors will have "throttle"?

Reply:
From offline discussion: It's to make falling back to higher levels easy